_content/av/beyond-distraction_shaila-catherine.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Beyond Distraction"
+authors:
+  - Shaila Catherine
+external_url: "https://wisdomexperience.org/wisdom-podcast/138-shaila-catherine/"
+drive_links:
+  - "https://drive.google.com/file/d/17khalkFYPwS2vhV8FDNWBpyecD7fSLic/view?usp=drive_link"
+course: problems
+tags:
+  - sati
+  - vipassana
```

Comment:
Is it vipassana? Feels applicable to samatha as well, no? If you meant the like American, "Insight" meditation community, perhaps the `american` tag?